plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -988,6 +986,7 @@ fu_cros_ec_usb_device_init(FuCrosEcUsbDevice *self)
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
 	fu_device_add_internal_flag(FU_DEVICE(self), FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID);
 	fu_device_set_version_format(FU_DEVICE(self), FWUPD_VERSION_FORMAT_TRIPLET);
+	fu_device_set_remove_delay(FU_DEVICE(self), CROS_EC_REMOVE_DELAY_RE_ENUMERATE);
```

Comment:
This changes the behavior though too.  It will apply to more devices.  The delay was only needed when `self->targ.common.flash_protection` was non-zero.

Reply:
>  It will apply to more devices

I think that's a good thing -- at the moment the remove_delay is unset for everything, and then for flash_protection devices we set it to re-enumerate. This PR changes it to have re-enumerate for everything rather than leaving it up for the daemon to guess. Note, the remove delay doesn't imply WAIT_FOR_REPLUG -- only setting the timeout to waif if we do set that flag.